pkg/plugin/plugin.go

```diff
@@ -123,7 +123,7 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 		return fmt.Errorf("failed to load netconf: %v", err)
 	}
 
-	utils.ConfigureLogging(conf.LogLevel)
+	utils.ConfigureLogging(conf.LogLevel, conf.LogFilePath)
```

Comment:
One consequence of this change is that all of the logs will be intertwined in the same log file (e.g. if multiple CNI plugins are running in parallel). 

Can you check to see if we are including the container identifiers in _every log line_ produced by the plugin?

e.g., containerID, interface name, networkName.

Reply:
We include either a container ID or a handle ID in most of the logs. From what I have found, the logs that do not include that information don't seem to need it. ex:
```
Attempting to load block cidr=192.168.92.0/26 host="matt-bz-0j96-kadm-node-1"